jekyll/_cci2/updating-server.adoc

```diff
@@ -53,6 +196,7 @@ replicatedctl app stop
 ```
 +
 Application shutdown takes a few minutes. Please check the administration dashboard and wait for the status to become “Stopped” before continuing. You can also run the following command to view the app status:
+// is this what we want them to do generally? In the step above we're saying to stop replicated from the dashboard
```

Comment:
the replicated command above is really the equivalent of pressing the `stop now` button in the UI. It doesn't stop replicated but only stops CircleCI. If we wanted to stop replicated we'd do `sudo service replicated stop` or something like that.